test/yubico_test.sh

```diff
@@ -0,0 +1,18 @@
+# Requires dfu-utils to be installed.
+# This runs through the steps described in https://github.com/Yubico/libu2f-host documentation
+set -x
+curl 'https://demo.yubico.com/wsapi/u2f/enroll?username=tomu&password=test' > regchallenge.json
+cat regchallenge.json
+echo "Touch key to confirm registration"
+u2f-host -a register -o 'https://demo.yubico.com'  < regchallenge.json > regdata.json
+#cat regdata.json
+curl https://demo.yubico.com/wsapi/u2f/bind -d "username=tomu&password=test&data=$(cat regdata.json)"
```

Comment:
can we generate random user/password here to avoid possible conflicts?

Reply:
Reasonable request.

Reply:
I've done this